homeassistant/components/history_stats/sensor.py

```diff
@@ -67,6 +74,12 @@ def exactly_two_period_keys(conf):
             vol.Optional(CONF_DURATION): cv.time_period,
             vol.Optional(CONF_TYPE, default=CONF_TYPE_TIME): vol.In(CONF_TYPE_KEYS),
```

Comment:
I believe you broke this line when CONF_TYPE_TIME became a list

This will impact sensory_type further below

Reply:
Thanks for chiming in, @dgomes . I'll address each of your concerns separately.

Please notice that `CONF_TYPE_TIME` **is not** a list. It is actually a string (namely, `time`), as defined in L41 of the same file. I think you got confused with its usage in the `UNITS` dict, in which it shows up as a key that has a list as value.